tools/MultiProjectValidator/ProjectLoader.cs

```diff
@@ -29,10 +30,23 @@ private static List<ProjectContext> LoadProjectContexts(string[] projectFiles)
             {
                 var fileTargetContexts = ProjectContext.CreateContextForEachTarget(file);
 
-                projectContexts.AddRange(fileTargetContexts);    
+                foreach(var context in fileTargetContexts)
+                {
+                    if (!TargetIsDesktop(context))
+                    {
+                        projectContexts.Add(context);
+                    }
+                }  
             }
 
             return projectContexts;
         }
+
+        private static bool TargetIsDesktop(ProjectContext context)
```

Comment:
This seems weird to be here. I think. This is called project loader, but it will skip one because it can't be validated. Should the validator be the one skipping it?! You could have an extension method in the project context, something like SkipProjectValidation where you could put this check.